pkg/kubernetes/deployer.go

```diff
@@ -0,0 +1,55 @@
+package kubernetes
+
+import (
+	"fmt"
+
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+)
+
+func (k *Kubernetes) GetDeployerResult(namespace string, pod *core.Pod) (string, error, string, error) {
```

Comment:
Thank you, Brave! 😍
But what I said yesterday, we must not have `kubernetes/deployer.go`. I'm sorry for my communication skills.

I think we can move this logic or function to `api/network.go`. 😄

Reply:
Maybe, I  understand your idea clearly. Let me give it a try!

Reply:
I updated